src/unix_term.rs

```diff
@@ -37,7 +36,7 @@ pub fn is_a_color_terminal(out: &Term) -> bool {
     }
 }
 
-pub fn c_result<F: FnOnce() -> libc::c_int>(f: F) -> io::Result<()> {
+fn c_result<F: FnOnce() -> libc::c_int>(f: F) -> io::Result<()> {
```

Comment:
Move the visibility changes into a separate commit as well? Ideally enable `warn(unreachable_pub)` so we can get them all at once.